streams/src/test/java/org/apache/kafka/streams/kstream/internals/suppress/KTableSuppressProcessorMetricsTest.java

```diff
@@ -149,7 +167,9 @@ public void shouldRecordMetrics() {
             );
 
         final MockInternalProcessorContext context = new MockInternalProcessorContext();
-        context.setCurrentNode(new ProcessorNode("testNode"));
+        final ProcessorNode processorNode = new ProcessorNode(TEST_NODE);
+        context.setCurrentNode(processorNode);
+        processorNode.init(context);
```

Comment:
We need to initialize the processor node in order to create the NodeMetrics so that it can be used for adding sensors later in the processor class, ditto in a few other places.